CourseTemplate/CourseTemplate.Core/CourseTemplate.Core.csproj

```diff
@@ -10,4 +10,10 @@
       <ProjectReference Include="..\CourseTemplate.Data\CourseTemplate.Data.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Reference Include="CustomExceptions">
+        <HintPath>..\..\CustomExceptions\bin\Debug\net7.0\CustomExceptions.dll</HintPath>
```

Comment:
No. It is very bad approach for dozens of reasons...

You should do the following:
Create solution AcademicHub.Common
Move exceptions there.
Open CourseTemplate solution.

![image](https://github.com/podkolzzzin/AcademicHub/assets/1740940/5089c146-77e1-426a-98c3-dd111b289de4)
Use add existing project and select `.csproj` of `AcademicHub.Common`